e2e-tests/experiments/infra-chaos/node_failure/run_e2e_test.yml

```diff
@@ -82,6 +82,9 @@ spec:
           - name: ZPOOL_NAME
             value: ''
 
+          - name: ZPOOL_ENCRYPTION_PASSWORD
+            value: 'test1234'
```

Comment:
@w3aman  is this value needed here in runner file?

Reply:
In case of encrypted zpool, after node-failure we have to again load this key into datasets so we need this value